pages/lecteur/cnaf/beneficiaire/index.js

```diff
@@ -167,16 +205,45 @@ export default function Beneficiaire() {
                 <td>{`${r.frequency} (${frequencyNames[r.frequency] || '?'})`}</td>
                 <td>{`${r.type} (${typeNames[r.type] || '?'})`}</td>
                 <td className={styles.numeric}>{r.total}</td>
+                <td className="shrink"><button onClick={handleRsaStats} data-index={i}>Afficher les stats</button></td>
                 <td className="shrink"><button onClick={handleDateHistogram} data-index={i}>Afficher par date</button></td>
                 <td>{r.error ? 'Oui' : 'Non'}</td>
               </tr>
             ))}
             </tbody>
           </table>
 
-          <button onClick={() => dispatchRuns({type: 'reset'})}>Vider l'historique</button>
+          <button onClick={() => dispatchRuns({type: 'reset'})} className={styles.margin_bottom}>Vider l'historique</button>
         </>)}
 
+
+        {rsaStats && <>
+          <h2 className={styles.subtitle}>
+            Statistiques droits & devoirs
+          </h2>
+
+          <table>
+            <thead>
+              <tr>
+                <th rowSpan="2">Dossiers</th>
+                <th rowSpan="2">Avec droits ouverts</th>
+                <th rowSpan="2">Avec devoirs ok</th>
+                <th rowSpan="2">Avec droits et devoirs ok</th>
+              </tr>
+              <tr>
+              </tr>
```

Comment:
```suggestion
              <tr>
                <th colSpan="N">Droits</th>
                <th colSpan="M">Devoirs/th>
              </tr>
              <tr>
              { keyDroits.map(k => (<th key=k>{label[k]]} ({k]})`))}
              { keyDevoirs.map(k => (<th key=k>{label[k]]} ({k]})`))}
              </tr>
```

Avec keyX qui reprend des infos de la doc:

> [...] Valeurs :'1'  [...] soumise à droits et devoirs [...] '0' [...] pas soumise à droits et devoirs


 

> Valeurs :0=Nouvelle demande en attente de décision CG pour ouverture du droit
1=Droit refusé
2=Droit ouvert et versable
3=Droit ouvert et suspendu (le montant du droit est calculable, mais l'existence du droit est remis en cause)
4=Droit ouvert mais versement suspendu (le montant du droit n'est pas calculable)
5=Droit clos
6=Droit clos sur mois antérieur ayant eu un contrôle dans le mois de référence pour une période antérieure.
[...]